jekyll.yml

```diff
@@ -164,3 +164,5 @@ mesh:
 
 # Version vars
 latest_gateway_oss_version: "3.9.1"
+
+konnect_konnectgatewaycontrolplane_api_version: "v1alpha2"
```

Comment:
Since this config file is global for all the documentation, I'd suggest to give this a prefix (like in https://github.com/Kong/developer.konghq.com/pull/2632/files#diff-db7658ee1b5ae4eeae1d7297aab0f327d9447cd6d9531ae4970d46d7f1f48c62R168) to designate that this is for the operator docs. WDYT?

Reply:
I used the API group as a prefix, but using the same prefix for all variables related to the operator docs is a good idea.